partner_compassion/models/partner_compassion.py

```diff
@@ -114,6 +118,14 @@ def get_unreconciled_amount(self):
     ##########################################################################
     @api.model
     def create(self, vals):
+        duplicate = self.search([
+                '|', '&', ('email', '=', vals.get('email')),
+                          ('email', '!=', False),
+                '&', '&', ('firstname', 'ilike', vals.get('firstname')),
+                          ('lastname', 'ilike', vals.get('lastname')),
+                          ('zip', '=', vals.get('zip'))])
+        duplicate_ids = [(4, itm.id) for itm in duplicate]
+        vals.update({'partner_duplicate_ids': duplicate_ids})
```

Comment:
If i put the filed not readonly in the .py and readonly in the view it doesn't work, so that the best option